src/outputs/textconfig.jl

```diff
@@ -100,3 +122,31 @@ end
 _rendertime!(img, config::Nothing, t) = img
 _rendertime!(img, config::TextConfig, t::Nothing) = img
 _rendertime!(img, config::Nothing, t::Nothing) = img
+
+
+@noinline _fontwrongtype(font) =
+    throw(ArgumentError(
+        """
+        font must be either a String or a FreeTypeAbstraction.FTFont,
+        got `$font` which is a `$(typeof(font))`.
+        """
+    ))
+
+@noinline _fontnotfounderror(font) =
+    throw(ArgumentError(
+        """
+        Font "$font" wasn't found in this system.
```

Comment:
Is there a reason these are wrapped so narrow now? 92 chars is the standard max width for Julia.

Reply:
This was an application of "one chunk of thought per line",
which makes things much easier to read for the documentation.
I agree that the newline before `throw` was unnecessary, a9dbfc809482a12d8ab598ade7bcd9f9b0247c8f is better ?